src/EventStore.Core/TransactionLog/Chunks/TFChunk/TFChunk.cs

```diff
@@ -673,13 +686,13 @@ private enum CacheStatus {
 		}
 
 		private void BuildCacheArray() {
-			var workItem = GetReaderWorkItem();
+			var workItem = AcquireFileReader();
```

Comment:
suggestion: `using var reader = AcquireFileReader()` and remove try-finally clause (+ also rename `workItem` to `reader`)

Reply:
i think it'd rather do this in another pr (although i could be talked into doing it in this pr in a separate commit) for the sake of the diff being as clear as possible because these changes are quite delicate

Reply:
i'm also ok with the change as-is, i mentioned it only for consistency reasons to make it similar to the way AcquireReader() is called in `VerifyFileHash()`